src/altinn-app-frontend/src/features/form/layout/index.ts

```diff
@@ -57,10 +57,34 @@ export type ComponentTypes =
   | 'Likert'
   | 'Panel';
 
-export interface IDataModelBindings {
-  [id: string]: string;
+export interface IDataModelBindingsSimple {
+  simpleBinding: string;
+}
+
+export interface IDataModelBindingsForGroup {
+  group: string;
+}
+
+/**
+ * A middle ground between group and simple bindings, a list binding can be used to
+ * store a list of primitive values, like string[].
+ */
+export interface IDataModelBindingsList {
+  list: string;
+}
+
+export interface IDataModelBindingsForAddress {
+  address: string;
+  zipCode: string;
+  postPlace: string;
 }
 
+export type IDataModelBindings =
```

Comment:
We are missing some types here if we want to go this route. For instance the `likert` binding for our new likert component.

Reply:
Huh, that should have produced typescript errors then? I haven't had any, and searching for `.likert` doesn't yield me any code accessing that property.

Reply:
Ah, you are absolutely right. In an earlier PR for the likert component we had a `likert` binding, but I see that this was changed. Also checked docs to be sure: https://docs.altinn.studio/app/development/ux/components/likert/

Sorry for the confusion :)